src/app/services/timeout/timeout.module.ts

```diff
@@ -0,0 +1,34 @@
+import { CommonModule } from "@angular/common";
+import { HTTP_INTERCEPTORS } from "@angular/common/http";
+import { NgModule, ModuleWithProviders } from "@angular/core";
+import {
+  TIMEOUT_OPTIONS,
+  TimeoutInterceptor,
+  TimeoutOptions,
+} from "./timeout.interceptor.service";
+
+@NgModule({
+  imports: [CommonModule],
+})
+export class TimeoutModule {
```

Comment:
Should probably still be namespaced. Especially if the module could ever be pulled out into its own repo